drivers/bbram/bbram_it8xxx2_emul.c

```diff
@@ -45,7 +45,7 @@ static int it8xxx2_emul_backend_get_data(const struct emul *target, size_t offse
 	return 0;
 }
 
-static const struct emul_bbram_backend_api it8xxx2_emul_backend_api = {
+static const struct emul_bbram_driver_api it8xxx2_emul_backend_api = {
```

Comment:
shouldn't you have an typedef for the old name and declared as obsolete, for some transition time?

Reply:
would it work with the `scripts/build/gen_kobject_list.py`, regardless, simple enough but this should probably go through Arch WG for stable API change

Reply:
the structure declaration is indeed public (see below),  the scope however is very thin. perhaps it's fine as it is.